docs/developer/options.md

```diff
@@ -0,0 +1,53 @@
+*Copied from https://gist.github.com/mootari/d39895574c8deacc57d0fc04eb0d21ca*
+*Original author: https://github.com/mootari*
+
+# Component options
```

Comment:
What about adding this to the `Options` TSDoc?

https://github.com/TypeStrong/typedoc/blob/1c4c5151cec938ef03cae79c33c12be8d587ea74/src/lib/utils/options/options.ts#L62-L67